episodes/5-transfer-learning.Rmd

```diff
@@ -0,0 +1,235 @@
+---
+title: "Transfer learning"
+teaching: 20
+exercises: 30
+---
+
+::: questions
+- How do I apply a pre-trained model to my data?
+:::
+
+::: objectives
+- Apply a state-of-the-art pre-trained network on your own dataset
+:::
+
+## What is transfer learning?
+Instead of training a model from scratch, with transfer learning you make use of models that are trained on another machine learning task. The pre-trained network captures generic knowledge about the domain during pre-training and will only be 'fine-tuned' to the specifics of your dataset.
+
+An example: Let's say that you want to train a model to classify images of different dog breeds. You could make use of a pre-trained network that learned how to classify images of dogs and cats. The pre-trained network will not know anything about different dog breeds, but it will have captured some general knowledge of, on a high-level, what dogs look like, and on a low-level all the different features (eyes, ears, paws, fur) that make up an image of a dog. Further training this model on your dog breed dataset is a much easier task than training from scratch, because the model can use the general knowledge captured in the pre-trained network.
+
+In this episode we will learn how use Keras to apply a state-of-the-art pre-trained model to the dollarstreet-10 dataset.
```

Comment:
I couldn't find a reference to the `-10` anywhere, have I missed it? Otherwise, I would propose to stick with the exact spelling to prevent confusions, including word splitting and capitalisation.

Also, how about adding a link to the dataset?

```suggestion
In this episode we will learn how use Keras to apply a state-of-the-art pre-trained model to the [Dollar Street Dataset](https://mlcommons.org/datasets/dollar-street/).
```

Reply:
Good point! You couldn't find it, because it is a dataset that I made myself. But I will call it Dollar Street Dataset, but link to https://zenodo.org/records/10970014 instead of the mlcommons dataset from which it is derived.